project_PassMeCash_Polkadot.md

```diff
@@ -0,0 +1,165 @@
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/onose-ogaga
+
+
+## Development Status :open_book:
+
+Details of the development of PassMeCash have been stated above with relevant links and we are working towards achieving our first MVP for this project, however we are to add some pages in the designs base on the recent flow diagram from the Team.
+
+
+## Development Roadmap :nut_and_bolt:
+
+This section should break the development roadmap down into milestones and deliverables. To assist you in defining it, we have created a document with examples for some grant categories [here](../docs/Support%20Docs/grant_guidelines_per_category.md). Since these will be part of the agreement, it helps to describe _the functionality we should expect in as much detail as possible_, plus how we can verify and test that functionality. Whenever milestones are delivered, we refer to this document to ensure that everything has been delivered as expected.
+
+Below we provide an **example roadmap**. In the descriptions, it should be clear how your project is related to Substrate, Kusama or Polkadot. We _recommend_ that teams structure their roadmap as 1 milestone ≈ 1 month.
+
+> :exclamation: If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Teams that submit others' work without attributing it will be immediately terminated.**
+
+### Overview
+
+- **Total Estimated Duration:** 12 weeks
+- **Full-Time Equivalent (FTE):** 3
+- **Total Costs:** 30,000 USD
+- 
+### Milestone 1 — Design Modification
+
+- **Estimated duration:** 3 weeks
+- **FTE:**  2
+- **Costs:** 5,000 USD
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | Modify the User Flow | The current user flow will be modified to accomodate the current project. |
+| 2. | Modify the User Persona | We will add more User Personas to the current ones as it will affect the PassMeCash-Polkadot. |
+| 3. | Modify the High Fidelity | More pages will be added to the High-Fidelity design. |
+| 4. | Modify the Prototype | Additional pages will be included to the prototype design too. |
```

Comment:
Can you expand these deliverables to include what you would actually be building for them? These should be concrete units of work that we can evaluate.

Reply:
before now PassMeCash was designed to solve the problem of requesting for cash via crypto, but with the recent cash policy enacted by the Central Bank Of Nigeria (CBN) which led to a serious challenge of cash scarcity that affected over 100 million Nigerians, thus we have to include into the existing design more flows and pages that will aid in the project development. The pages to be added include:

1. Account locked page
{page to display account locked}

2. Unlock account page
{shows security questions, phone numbers for OTP and transfer PIN}
Reset password
Forgot password

3. Transfer page 
{page to contain 
- send funds to PMC user
- send bulk funds to PMC user 
- send funds to external bank
- send to bulk external bank
- scan QR code
- add beneficiary 
	{show added beneficiary on same page} 
- schedule payment 
- generates invoices

4. Receive page
- Receive account page 
	{displays account number & tag (username)} 
- generates QR Code 

5. Crypto Dashboard 
- detail page for individual coin
		{transactions history of coin & receive, swap button = pop up} 
view available agent in list view

6. Settings
- charge password
- PassMeCash PIN management 
- security questions 
- session page 
- appearance 
	{dark mode, hide balance & auto swap} 
/edit/

7. Profile page
{displays profile picture, tag (username), account number, email, sign up date, address, gender, DOB, phone number, BVN}

Reply:
Please add this information to the document, as it will be much easier to find this information when evaluating milestones.

Reply:
@semuelle 
These information are now added to the document as you suggested. Thank you.